server/controllers/lecture-controller.js

```diff
@@ -120,7 +123,7 @@ exports.newBooking = async (req, res) => {
             // Get the lecture information
             const lectureQuery = knex
               .select(
-                { name: "lecture.name" },
+                //{ name: "lecture.name" },
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.